README.md

```diff
@@ -51,7 +51,8 @@ for status in client.register(store):
 # Keep the 'store' object because it contains now the access token
 # and use it next time you want to register on the TV.
 print(store)
-# {'client_key': 'ACCESS_TOKEN_FROM_TV'}
```

Comment:
This line was actually supposed to be the output of print(..) statement above.

To me, saying `store = { ..}` on the line below adds to the confusion. Further, line 25 is the answer to the question how to use "store". If you feel it is lacking, feel free to send across a PR to make it less confusing. :)

As of this line, I think it adds to the confusion.

Reply:
👀  on this if it helps get rid of the confusion. Else, feel free to close the PR :)